spec/features/5_photos_spec.rb

```diff
@@ -1,120 +1,126 @@
 require "rails_helper"
 
-show_tests_in_browser = false
-do_not_show_tests_in_browser = false
+feature "Photos:" do
+
+  context "photos show page" do
+    before(:each) do
+      @user_1 = FactoryGirl.create(:user, :username => "1", :email => "1@m.com")
```

Comment:
How about using factories with sequences to clean this up? Since factories should be [minimally viable out of the box](https://robots.thoughtbot.com/factories-should-be-the-bare-minimum)

Reply:
https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#sequences